dll/win32/iernonce/registry.cpp

```diff
@@ -0,0 +1,186 @@
+/*
+ * PROJECT:     ReactOS system libraries
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Functions to read RunOnceEx registry.
+ * COPYRIGHT:   Copyright 2021 He Yang <1160386205@qq.com>
+ */
+
+
+#include <cassert>
+#include "registry.h"
+#include <winreg.h>
+#include <cstdlib>
```

Comment:
```suggestion
#include <cassert>
#include <cstdlib>
#include "registry.h"
#include <winreg.h>
```

Reply:
Oh yes, I forgot to sort these headers, my mistake 🤣
But... shouldn't we put system header files (`<>`) in front of user-defined headers (`""`)

Reply:
Yes the system header files should go before the user-defined ones.

Reply:
I fixed these and some other mistakes in another commit!